pkg/maps/lbmap/lbmap.go

```diff
@@ -187,6 +190,44 @@ func (*LBBPFMap) DeleteBackendByID(id uint16, ipv6 bool) error {
 	return nil
 }
 
+// DeleteAffinityMatch removes the affinity match for the given svc and backend ID
+// tuple from the BPF map
+func (*LBBPFMap) DeleteAffinityMatch(revNATID uint16, backendID uint16) error {
+	return AffinityMatchMap.Delete(
+		NewAffinityMatchKey(revNATID, uint32(backendID)).ToNetwork())
+}
+
+// AddAffinityMatch adds the given affinity match to the BPF map.
+func (*LBBPFMap) AddAffinityMatch(revNATID uint16, backendID uint16) error {
+	return AffinityMatchMap.Update(
+		NewAffinityMatchKey(revNATID, uint32(backendID)).ToNetwork(),
+		&AffinityMatchValue{})
+}
+
+// DumpAffinityMatches returns the affinity match map represented as a nested
+// map which first key is svc ID and the second - backend ID.
+func (*LBBPFMap) DumpAffinityMatches() (map[uint16]map[uint16]struct{}, error) {
+	matches := map[uint16]map[uint16]struct{}{}
```

Comment:
Suggestion: might be nice to convert these map types into new types so that they're more readable.